openapi.yaml

```diff
@@ -0,0 +1,787 @@
+      responses:
+        "200":
+          description: "Display created Securable"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/SecurableResponse"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        "500":
+          $ref: "#/components/responses/ServerError"
+  /securables/{securableId}:
+    parameters:
+      - name: "securableId"
+        in: "path"
+        description: "Id of Securable to return"
+        required: true
+        schema:
+          type: string
+    get:
+      operationId: readSecurable
+      tags:
+        - "securables"
+      responses:
+        "200":
+          description: "Display Securable"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/SecurableResponse"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        "500":
+          $ref: "#/components/responses/ServerError"
+    put:
+      tags:
+        - "securables"
+      operationId: updateSecurable
+      requestBody:
+        required: true
+        content:
+          'application/json':
+            schema:
+              $ref: "#/components/schemas/Securable"
+      responses:
+        "200":
+          description: "Display updated Securable"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/SecurableResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        "500":
+          $ref: "#/components/responses/ServerError"
+  /securables/{securableId}/grants:
+    get:
+      tags:
+        - "securables"
+        - "grants"
+      operationId: readGrantsForSecurable
+      parameters:
+        - name: "securableId"
+          in: "path"
+          description: The securable to display grants for
+          required: true
+          schema:
+            type: string
+        - name: "subject"
+          in: "query"
+          description: Limit the assignments to the specified subject, optional
+          schema:
+            type: string
+        - $ref: "#/components/parameters/pagingOffset"
+        - $ref: "#/components/parameters/pagingLimit"
+      responses:
+        "200":
+          description: "Display Grants for Securable"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/GrantsResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "500":
+          $ref: "#/components/responses/ServerError"
+  /check-permissions:
+    parameters:
+      - name: "securableIds"
+        in: query
+        description: Id of the securable
+        required: true
+        schema:
+          type: array
+          items:
+            type: string
+          minItems: 1
+      - name: "permissions"
```

Comment:
What is a "permission" in this context?  It should be a permission plus a securable right?